parser.go

```diff
@@ -110,6 +110,26 @@ var (
 	pString = newTokenPattern("string", "\".*\"")
 )
 
+func (p *parser) parseWorksheets() (map[string]*tWorksheet, error) {
+	wsDefs := make(map[string]*tWorksheet)
+
+	for p.peek() == pWorksheet.name {
```

Comment:
Yes, though arguably, you should write this as `for pWorksheet.re.MatchString(p.peek()) {`